production-swarm/docker-stack.yml

```diff
@@ -0,0 +1,58 @@
+version: '3'
+services:
+  jobservice:
+    env_file:
+      - ./rabbitmq.env
+      - ./postgres.env
+      - ./jobservice.env
+    environment:
+      CAF_DATABASE_USERNAME: "${CAF_DATABASE_USERNAME:-postgres}"
+      CAF_DATABASE_PASSWORD: "${CAF_DATABASE_PASSWORD:-root}"
+      CAF_STATUS_CHECK_TIME: 5
+      CAF_TRACKING_PIPE: "${CAF_TRACKING_PIPE_IN:-jobtracking-in}"
+      CAF_WEBSERVICE_URL: http://jobservice:9411/job-service/v1
+    image: jobservice/job-service:latest
+    deploy:
+      restart_policy:
+        condition: any
+        delay: 5s
+        max_attempts: 3
+        window: 120s    
+      mode: replicated
+      replicas: 2
+      resources:
+        limits:
+          cpus: '0.5'
+          memory: 1024M
+      update_config:
+        parallelism: 1
+        delay: 10s
+    ports:
+      - "9411:8080"
+
+  worker-jobtracking:
+    env_file:
+      - ./rabbitmq.env
+      - ./postgres.env      
+    environment:
+      CAF_WORKER_INPUT_QUEUE: "${CAF_TRACKING_PIPE_IN:-jobtracking-in}"
```

Comment:
These are not configurable in marathon I see. I'm happy to make them none configurable here also